src/foremast/utils/subnets.py

```diff
@@ -42,11 +76,19 @@ def get_subnets(
         region (str): AWS Region to find Subnets for.
 
     Returns:
```

Comment:
I think this refactored function should just return 1 'format'.
Maybe
```
{'account': 'x', 'region': ['az']}
```

Or maybe this is split into two functions:
`get_all_subnets()`, `get_subnet(account=x)`

Reply:
Yes, I noticed that TODO. Will work that in somehow.